### PR TITLE
Add support for volume capabilities endpoint

### PR DESCRIPTION
**Pull request by cpuguy83**
Signed-off-by: Brian Goff <cpuguy83@gmail.com>

### CODE REVIEW

**Comment by calavera** [comment]
LGTM

File: volume/api.go [hunk 1]
```diff
@@ -10,28 +10,31 @@ const (
...
+	mountPath        = "/VolumeDriver.Mount"
+	unmountPath      = "/VolumeDriver.Unmount"
+	capabilitiesPath = "/VolumeDriver.Capabilities"
 )
 
 // Request is the structure that docker's requests are deserialized to.
 type Request struct {
 	Name    string
 	Options map[string]string `json:"Opts,omitempty"`
+	MountID string            `json:"omitempty"`
```
Comment (runcom) [inline volume/api.go]: Missing comma before omitemtpy

Comment (cpuguy83) [inline volume/api.go]: Nice catch!

Comment (cpuguy83) [inline volume/api.go]: Updated

**Comment by runcom** [comment]
LGTM will merge after tests

File: volume/api.go [hunk 2]
```diff
@@ -10,28 +10,31 @@ const (
...
+	mountPath        = "/VolumeDriver.Mount"
+	unmountPath      = "/VolumeDriver.Unmount"
+	capabilitiesPath = "/VolumeDriver.Capabilities"
 )
 
 // Request is the structure that docker's requests are deserialized to.
 type Request struct {
 	Name    string
 	Options map[string]string `json:"Opts,omitempty"`
+	MountID string            `json:"MountID,omitempty"`
```
Comment (runcom) [inline volume/api.go]: should not need the json tag since MountID is exported

Comment (cpuguy83) [inline volume/api.go]: ok

Comment (runcom) [inline volume/api.go]: (sorry for the nitpicking, I've just noticed this - was going to merge anyway)

**Pull request merged**